packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -426,13 +426,16 @@ function BackgroundSizeToolsPanelItem( {
 	const updateBackgroundSize = ( next ) => {
 		// When switching to 'contain' toggle the repeat off.
 		let nextRepeat = repeatValue;
+		let nextPosition = positionValue;
 
 		if ( next === 'contain' ) {
 			nextRepeat = 'no-repeat';
+			nextPosition = undefined;
```

Comment:
Setting to `undefined` between toggles so that `"50% 0"` doesn't persist.

This means however, that any user-defined position values are lost as well.

I'm wondering if we should save the value in local state and only reset when a default value is detected, e.g, `"50% 0"`

Reply:
IMO it's fine to reset the position when switching between sizes as you're kind of resetting how the image will appear anyway. Could be a follow-up PR if we think it's needed after all?